server/api/src/rs_get_rescinfo_for_get.cpp

```diff
@@ -0,0 +1,94 @@
+#include <cstring> // for strdup.
+#include <nlohmann/json.hpp> // for strdup.
```

Comment:
As a convention, we include C++ stdlib headers last (in their own group).

Reply:
strdup duplicate removed